### PR TITLE
Release v1.10.0

### PR DESCRIPTION
**Pull request by jmrodri**
Signed-off-by: jesus m. rodriguez <jesusr@redhat.com>

### CODE REVIEW

File: changelog/generated/v1.10.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Provide XML formatting option for scorecard users. Additionally transforms scorecard result types to xunit testsuite/testcase layout. ([#5048](https://github.com/operator-framework/operator-sdk/pull/5048))
+
+### Changes
+
+- **Breaking change**: (ansible/v1) Bumped Kustomize version in ansible project scaffolding to 3.8.7. ([#5057](https://github.com/operator-framework/operator-sdk/pull/5057))
+
+### Bug Fixes
+
+- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dcokerfile. ([#5030](https://github.com/operator-framework/operator-sdk/pull/5030))
```
Comment (varshaprasad96) [inline changelog/generated/v1.10.0.md:13]:
```suggestion
- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dockerfile. ([#5030](https://github.com/operator-framework/operator-sdk/pull/5030))
```

File: website/content/en/docs/upgrading-sdk-version/v1.10.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: v1.10.0
+weight: 998990000
+---
+
+## (ansible/v1) Bump Kustomize version in helm project scaffolding to 3.8.7
+
+Ansible projects are now scaffolded with `Kustomize` version `3.8.7`. This version of kustomize requires that the path to patch file be provided with the `--path` flag in `add patch` command. In `molecule/default/prepare.yml` file, update the command under `Set pull policy`, to be: ```diff
+     - name: Set pull policy
+-      command: '{{ "{{ kustomize }}" }} edit add patch pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' +      command: '{{ "{{ kustomize }}" }} edit add patch --path pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' ```
```
Comment (varshaprasad96) [inline website/content/en/docs/upgrading-sdk-version/v1.10.0.md:10]:
```suggestion
Ansible projects are now scaffolded with `Kustomize` version `3.8.7`. This version of kustomize requires that the path to patch file be provided with the `--path` flag in `add patch` command. In `molecule/default/prepare.yml` file, update the command under `Set pull policy`, to be: 
```diff
     - name: Set pull policy
-      command: '{{ "{{ kustomize }}" }} edit add patch pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' 
+      command: '{{ "{{ kustomize }}" }} edit add patch --path pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' ```
```
this should be appearing as a diff. maybe formatting issue

**Comment by laxmikantbpandhare** [comment]
@jmrodri - Looks like testdata is missing.

File: changelog/generated/v1.10.0.md (hunk 1 above)
Comment (jmrodri) [reply changelog/generated/v1.10.0.md:13]: Ugh I thought I fixed that.

**Comment by jmrodri** [comment]
@varshaprasad96 @laxmikantbpandhare fixed both items. I will post a follow on PR to fix the release doc since it does not mention test-sanity for major/minor releases

**Comment by laxmikantbpandhare** [comment]
> @varshaprasad96 @laxmikantbpandhare fixed both items. I will post a follow on PR to fix the release doc since it does not mention test-sanity for major/minor releases

Yes, we need to mention in the doc that `test-sanity` does not work in the release doc.

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

**Pull request merged**